app/views/provider/admin/user/notification_preferences/_form.html.slim

```diff
@@ -6,11 +6,10 @@
       legend = t(category.title_key, scope: :notification_category_titles)
 
       ol
-
         = f.input :enabled_notifications, as: :check_boxes_with_hints,
-                                          collection: category.notifications,
-                                          member_label: ->(value) { t(value, scope: :notification_preference_titles) },
-                                          member_hint: ->(value) { t(value, scope: :notification_preference_hints) }
+                                  collection: category.notifications.map { |value| [t(value, scope: :notification_preference_titles), value] },
+                                  member_hint: ->(value) { t(value, scope: :notification_preference_hints) }
+
```

Comment:
Here is an extra line that can be removed.

Reply:
done